tools/static-code-analysis/checkstyle/suppressions.xml

```diff
@@ -19,4 +19,5 @@
     <!-- Some source files have different headers -->
     <suppress files=".+org.eclipse.smarthome.automation.+" checks="HeaderCheck"/>
     <suppress files=".+org.eclipse.smarthome.config.dispatch.test.+" checks="ServiceComponentManifestCheck"/>
+	<suppress files=".+org.eclipse.smarthome.core.thing.+" checks="PackageExportsNameCheck"/>
```

Comment:
could you please use spaces instead of a tab to have it consistent with the lines above?